common/common/src/main/java/io/helidon/common/FeatureCatalog.java

```diff
@@ -152,6 +154,10 @@
                     .path("WebServer", "Websocket")
                     .nativeSupported(true)
                     .nativeDescription("Server only"));
+        addSe("io.helidon.faulttolerance",
+              "Fault Tolerance",
+              "Fault tolerance support",
+              "FT");
```

Comment:
intentional?

Reply:
If so its missing in the [docs/se/aot/01_introduction.adoc](https://github.com/tomas-langer/helidon/blob/1294-grpc-native/docs/se/aot/01_introduction.adoc)

Reply:
Removed